app/templates/serviceLearning/slcQuestionnaire.html

```diff
@@ -1,48 +1,50 @@
 {%set title = "SLC Questionnaire"%}
 {% set isViewSlc = 'view' in request.path %}
 
+<link rel="stylesheet" href="{{url_for('static', filename ='css/slcQuestionnaire.css')}}">
```

Comment:
This needs to be added into the styles block, not straight into the template.

Reply:
resolved